frontend/app/quest/Quest_Activity.html

```diff
@@ -0,0 +1,107 @@
+<!DOCTYPE html>
+    <!-- to do:  update central stylesheet to this style sheet code.-->
+    
+<html ng-app="quest_builder">
+    <head>
+        <meta charset="utf-8">
+        <meta name="description" content="Hello">
+        <meta name="viewport" content="width=device-width,initial-scale=1">
+        <link rel="shortcut icon" href="https://s3-us-west-2.amazonaws.com/itreeware/PL_Favicon.png" type="image/x-icon">
+        <link rel="icon" href="https://s3-us-west-2.amazonaws.com/itreeware/PL_Favicon.png" type="image/x-icon">
+        <title>PlanetLAB> Quest Builder</title>
+        <link rel="stylesheet" href="css/style.css" type="type/css" />
+        </head>
+
+    <body>
+    <div ng-view></div>
+        <img src="https://s3-us-west-2.amazonaws.com/itreeware/PL--Header.png" alt="PlanetLAB |Freedom Games">
+        <ol class="controlNav" >
+            <li class="questControlNav"> 
+                <a  href="Quest_Basic.html">Basic Info</a>
+            </li>
+            <li class="questControlNav"> 
+                <a  href="Quest_Activity.html">Activity Details</a>
+            </li>
+            <li class="questControlNav"> 
+                <a  href="Quest_for teachers.html">For Teachers</a>
+            </li>
+            <li class="questControlNav"> 
+                <a  href="Quest_Review.html">Review</a>
+            </li>
+        </ol>
+ <h1>Quest: Activity Details</h1>       
+<form>
+<fieldset>
+	<section class="row">
+Duration: <input type="text" Duration="Duration" > <!--format HH:mm:ss-->
+	</section>
+
+<section class="row">
+Material &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
```

Comment:
What's the goal for these nbsp's? We may be able to solve the original issue through CSS styling